tensorflow/compiler/xla/mlir/runtime/transforms/BUILD

```diff
@@ -231,7 +231,23 @@ cc_library(
         "@llvm-project//mlir:Parser",
         "@llvm-project//mlir:Pass",
         "@llvm-project//mlir:ToLLVMIRTranslation",
-    ],
+    ] + select({
+        "//tensorflow:arm_any": [
+            "@llvm-project//llvm:AArch64AsmParser",
+        ],
+        "//tensorflow:linux_ppc64le": [
+            "@llvm-project//llvm:PowerPCAsmParser",
+        ],
+        "//tensorflow:macos_arm64": [
+            "@llvm-project//llvm:AArch64AsmParser",
+        ],
+        "//tensorflow:linux_s390x": [
+            "@llvm-project//llvm:SystemZAsmParser",
```

Comment:
Unfortunately, internally we don't support SystemZ. So I can only merge this PR without this select statement.

Reply:
Thanks for pointing to this :) I am not working on that part of the code, so I didn't know this. Do you want to create another PR for this?

Reply:
Done: https://github.com/tensorflow/tensorflow/pull/59493

Seems like the defines have been moved down to `tensorflow/tsl` so changed that as well compared to the initial PR.